packages/source-airtable/index.js

```diff
@@ -0,0 +1,23 @@
+const Airtable = require('airtable')
+
+module.exports = function (api, options) {
+  const base = new Airtable({ apiKey: options.apiKey }).base(options.base)
+
+  api.loadSource(async store => {
+    const contentType = store.addContentType({
+      typeName: options.typeName,
+      route: `/${options.route}/:id`
```

Comment:
```suggestion
      route: options.route
```
I think we can make routing optional :)

Reply:
Done.